Ewso365 fix fetch incidents (#13750)
demisto/content: Packs/EWS/ReleaseNotes/1_8_28.md

#### Integrations
##### EWS O365

> - Fixed an issue when fetching incidents with lower max_fetch value then number of incidents with same creation time. 
> ##### EWS v2
> - Fixed an issue when fetching incidents with lower max_fetch value then number of incidents with same creation time. 

Reviewer: ```suggestion
- Fixed an issue where fetching incidents with the same creation time missed to retrieve some of them. 
##### EWS v2
- Fixed an issue where fetching incidents with the same creation time missed to retrieve some of them. 
```
